gamestonk_terminal/screener/presets/rosenwald.ini

```diff
@@ -0,0 +1,25 @@
+# Author of preset: Traceabl3
+# Contact: three shots in the air.
+# Description: the classic rosenwald screen based on some dude i work withs best guess.
```

Comment:
The test is failing on the "withs" here, should be "with"

Reply:
Fixed the preset template, and updated the read me file.